introduction/report-structure.md

```diff
@@ -0,0 +1,18 @@
+# Report structure
+To anwser the four main subquestions, the report will have four sections.  
+The first subquestion zooms in on what can be learned from (non-software) communities.
+Before theories can be defined about what can be learned from all sorts of communities, it must be defined what a community is.
+Because communities exist in different sizes, with different structures and on different platforms, a few of these communities will be analysed in section 1.
+
+Software development in its current form has some methodologies on how requirements are established for a (big) software development project.
+*** referentie naar subvraag ***  
```

Comment:
If we rephrase 'How can community resources be used to improve the software development process?' to something like 'How can the software development process be structured to include community contributions?', we can reference the question here by saying: 'In order to include community contributions in such a project, the methodology has facilitate it.' Or something like that